src/pint/fitter.py

```diff
@@ -135,7 +135,9 @@ def __get__(self, instance, owner=None):
                 )
             try:
                 cache = instance.__dict__
-            except AttributeError:  # not all objects have __dict__ (e.g. class defines slots)
+            except (
+                AttributeError
+            ):  # not all objects have __dict__ (e.g. class defines slots)
```

Comment:
Would it help to put the comment on a new line to avoid the ugly bracketing?

Reply:
Yes. I'll do that.